homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -66,9 +73,17 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     incl_filter = config.get(CONF_INCL_FILTER)
     excl_filter = config.get(CONF_EXCL_FILTER)
     realtime = config.get(CONF_REALTIME)
+    vehicle_type = config.get(CONF_VEHICLE_TYPE)
+    units = config.get(CONF_UNITS)
```

Comment:
`config.get(CONF_UNITS, hass.config.units.name)`

Reply:
In this platform, the units can be overridden.  Later down in the code the units are grabbed if the default isn't populated.

    if units is None:
        units = hass.config.units.name